test/src/test/groovy/hudson/model/AbstractProjectTest.groovy

```diff
@@ -594,4 +596,37 @@ public class AbstractProjectTest extends HudsonTestCase {
         }
         return con
     }
+
+    @Issue("JENKINS-27549")
+    public void testLoadingWithNPEOnTriggerStart() {
+        AbstractProject project = jenkins.createProjectFromXML("foo", getClass().getResourceAsStream("AbstractProjectTest/npeTrigger.xml"))
+
+        assert project.triggers().size() == 1
+    }
+
+    @TestExtension
+    static class MockBuildTriggerThrowsNPEOnStart<Item> extends Trigger {
+        @Override
+        public void start(hudson.model.Item project, boolean newInstance) { throw new NullPointerException(); }
+
+        @Override
+        public TriggerDescriptor getDescriptor() {
+            return DESCRIPTOR;
+        }
+
+        @TestExtension
+        public static final TriggerDescriptor DESCRIPTOR = new DescriptorImpl()
```

Comment:
Delete this line. It is deprecated to put `@[Test]Extension` on a field. Should be on the class.